auth.js

```
@@ -208,6 +212,11 @@ async function main() {
     await dbClient.end();
   }
 
+  console.log("Performance test run with a total of " + failCount + " metrics over budget.")
+  if (FAIL_BUDGETS && failCount > 0) {
+    process.exit(1);
+  }
+
```

I would suggest we move the exit status handling within the one invoking the main function. Also, maybe better to raise an exception here then catch it outside. So we can catch all exceptions and return exit status 1 not just for failed budgets, but for all exceptions.